### PR TITLE
Update Min supported version to iOS 11, tvOS 11 and Xcode 14.1  + Extension version Bump to 4.0.0

### PR DESCRIPTION
**Pull request by addb**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by kevinlind — APPROVED** [review]
👍

File: Makefile [hunk 1]
```diff
@@ -9,38 +9,69 @@ IOS_SIMULATOR_ARCHIVE_PATH = $(CURR_DIR)/build/ios_simulator.xcarchive/Products/
...
 # pod repo update may fail if there is no repo (issue fixed in v1.8.4). Use pod install --repo-update instead
 pod-install:
-	(pod install --repo-update)
+	pod install --repo-update
+
+pod-update: pod-repo-update
+	pod update
 
 ci-pod-install:
 	(bundle exec pod install --repo-update)
```
Comment (emdobrin) [inline Makefile:40]: nit: can you remove parenthesis here too

**Review by emdobrin — APPROVED** [review]

**Review by emdobrin — APPROVED** [review]

**Review by emdobrin — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -9,38 +9,69 @@ IOS_SIMULATOR_ARCHIVE_PATH = $(CURR_DIR)/build/ios_simulator.xcarchive/Products/
...
 TEST_APP_TVOS_SCHEME = TestApptvOS
 
 setup:
-	(pod install)
+	pod install
 
 setup-tools: install-githook
 
+clean:
+	rm -rf build
```
Comment (cacheung) [inline Makefile:27]: Is it rm -rf ./build or rm-rf build?

File: Makefile [hunk 3]
```diff
@@ -105,16 +113,16 @@ install-githook:
 	./tools/git-hooks/setup.sh
 
 lint-autocorrect:
-	(./Pods/SwiftLint/swiftlint  autocorrect --format)
+	./Pods/SwiftLint/swiftlint  autocorrect
```
Comment (cacheung) [inline Makefile:116]: nit: Is there an extra space before autocorrect?

**Pull request merged**